app/test/test-support/src/main/java/io/syndesis/test/container/s2i/S2iProjectBuilder.java

```diff
@@ -48,7 +50,21 @@ public Project build(IntegrationSource source) {
         SyndesisS2iAssemblyContainer syndesisS2iAssemblyContainer = new SyndesisS2iAssemblyContainer(integrationName, projectDir, imageTag);
         syndesisS2iAssemblyContainer.start();
 
-        Path fatJar = projectDir.resolve("target").resolve("project-0.1-SNAPSHOT.jar");
+        // The S2I assembly container result need to be copied to the local host
+        // to be used by S2I integration containers
+        Path target = projectDir.resolve("target");
+        Path fatJar = target.resolve("project-0.1-SNAPSHOT.jar");
+        String s2iAssemblyContainerSourcePath = "/tmp/src/target/project-0.1-SNAPSHOT.jar";
```

Comment:
This is very Linux dependent. Maybe use one of the Files.createTmpDir utilities out there?

(Probably not problematic for our continuous delivery, but bugging to some developers?)

Reply:
+1

Reply:
From what I understand this is the path within the container, doesn't represent the local filesystem path. And it's fixed as Java S2I build runs Maven within `/tmp/src`. Not sure why does this have to be a variable, could be fed directly to `copyFileFromContainer`.

Reply:
Aah, didn't notice it runs inside the container. I just have an automatic alarm everytime I see an absolute path on the code.

Should be fine then.

Reply:
> From what I understand this is the path within the container, doesn't represent the local filesystem path. And it's fixed as Java S2I build runs Maven within `/tmp/src`. Not sure why does this have to be a variable, could be fed directly to `copyFileFromContainer`.

As I am using the path again in the exception I prefer not to hard copy the path and rely on a variable instead